core-web/apps/dotcms-ui/src/app/view/components/_common/dot-custom-time.component/dot-custom-time.component.ts

```diff
@@ -1,28 +1,11 @@
-import { AfterViewChecked, Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
-
-import { DotFormatDateService } from '@dotcms/app/api/services/dot-format-date-service';
+import { Component, Input, ViewEncapsulation } from '@angular/core';
 
 @Component({
     encapsulation: ViewEncapsulation.Emulated,
     selector: 'dot-custom-time',
     styleUrls: ['./dot-custom-time.component.scss'],
     templateUrl: 'dot-custom-time.component.html'
```

Comment:
Could you add to this component  ` changeDetection: ChangeDetectionStrategy.OnPush,`

Reply:
Alright, I'll do it!